### PR TITLE
Feature/ultralytics all models

### PR DESCRIPTION
**Pull request by hardikdava**
This PR is related to #221 

## Type of change

Please delete options that are not relevant.

-   [x] New feature (non-breaking change which adds functionality)
-   [x] This change requires a documentation update

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

**Comment by hardikdava** [comment]
Hello @SkalskiP :wave: , I finished this feature. Please test it with this colab: https://colab.research.google.com/drive/1PST2Yv56IjmKvn6J0I3JCGMT-IgDTEC_?usp=sharing

I have added tests for all the models and deprecated for existing api `sv.Detections.from_yolov8()`. Let me know if you have any trouble with tests.

**Comment by SkalskiP** [comment]
Hi, @hardikdava 👋🏻! Awesome, let's have it ready for the `supervision-0.13.0` release.

File: supervision/dataset/core.py [hunk 1]
```diff
@@ -46,6 +46,14 @@ def split(
     ) -> Tuple[BaseDataset, BaseDataset]:
         pass
 
+    def add_instance(
```
Comment (SkalskiP) [inline supervision/dataset/core.py:49]: @hardikdava looks to me like this change is not used.

**Review by hardikdava — COMMENTED** [review]
Does it require any further modification?

File: supervision/dataset/core.py (hunk 1 above)
Comment (hardikdava) [reply supervision/dataset/core.py]: Sorry, it was a mixed up in commits. But it is not needed so I removed it.

**Comment by SkalskiP** [comment]
@hardikdava I tested your notebook. It works, and it is fantastic! Merging.

**Pull request merged**